how-to.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: two-col
+---
+
+<div class="margin-trailer">
+  <h2>What to Share</h2>
+  <p>
+    Stories submitted to The Second Page should be largely true, and involve your own experiences.
+    We welcome stories of all sizes (a few minutes up to half an hour). Recordings are preferred in WAV or AIF format,
+    though MP3s are okay. Recordings should be read aloud by you or a friend.
+  </p>
+  <p>
+    If you would like to contribute, but cannot read your story or prefer to remain anonymous, let us know.
+    If aired, we can likely include your work under another name, and record it ourselves.
+  </p>
+  <h2>How to Share</h2>
+  <p>
+    We're pretty accomodating with regard to sending/receiving your recordings. However, we recommend
+    link sharing using a service like
+    <a href="drive.google.com" target="_blank" rel="noopener noreferrer">Google Drive</a>
+    or
+    <a href="https://www.dropbox.com" target="_blank" rel="noopener noreferrer">Dropbox</a>.
+  </p>
+  <p>
+    To send a link to your recording, or if you have other questions, email us directly at either
+    <a href="mailto:harris@chromamine.com">harris@chromamine.com</a> for Harris or
+    <a href="mailto:smh@seanmhanson.com">smh@seanmhanson.com</a> for Se&aacute;n.
+    For sufficiently small files, you may wish to email us the file directly.
+  </p>
+  <h2>Tips on Recording</h2>
+  <p>
+    (forthcoming)
```

Comment:
This will likely be a page in and of itself. I want to give some advice about recording on a budget, tips at home, etc.